src/components/ont/OntFlowcell.vue

```diff
@@ -32,36 +34,32 @@ export default {
       required: true
     }
   },
-  data () {
-    return {
-      hover: false
-    }
-  },
   methods: {
-    updateFlowcell (libraryName) {
-      this.$emit('updateFlowcell', this.position, libraryName)
-    },
-    allowDrop (event) {
-      event.preventDefault()
-      this.hover = true
-    },
-    endDrop (event) {
-      event.preventDefault()
-      this.hover = false
-    },
-    drag (name, event) {
+    drag (event) {
       if (name === 0) return
       const img = new Image()
       img.src = '/tube.png'
       event.dataTransfer.setDragImage(img, 80, 0)
       event.dataTransfer.setData('flowcellPosition', this.position)
+      event.dataTransfer.setData('libraryName', this.library.name)
+
+      event.dataTransfer.setData('sourceType', 'FLOWCELL')
+
       this.hover = false
     },
     drop (event) {
       event.preventDefault()
-      this.updateFlowcell(event.dataTransfer.getData('name'))
+      let libraryName = event.dataTransfer.getData('libraryName')
+      this.updateFlowcell(this.position, libraryName)
+
+      if (event.dataTransfer.getData('sourceType') === 'FLOWCELL') {
```

Comment:
why do you have to do this?

Reply:
It's for drag and drop from one flowcell to another. Depending if the flowcell got dragged from the library list/ flowcell, it needs to update different things